eng/native/configurecompiler.cmake

```
@@ -645,6 +645,13 @@ if (CLR_CMAKE_HOST_UNIX)
     endif()
   endif()
 
+  if (CMAKE_CXX_COMPILER_ID)
+    check_cxx_compiler_flag(-Wno-nontrivial-memaccess COMPILER_SUPPORTS_FNO_NONTRIVIAL_MEMACCESS)
+    if (COMPILER_SUPPORTS_FNO_NONTRIVIAL_MEMACCESS)
```

nit: we can check the positive flag and then negate it:
```suggestion
  if (CMAKE_CXX_COMPILER_ID)
    check_cxx_compiler_flag(-Wnontrivial-memaccess COMPILER_SUPPORTS_W_NONTRIVIAL_MEMACCESS)
    if (COMPILER_SUPPORTS_W_NONTRIVIAL_MEMACCESS)
```

also please move it after line 622 in clang specific block.